main.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 func main() {
-	createDatabase := flag.Bool("createDatabase", false, "true if you need casdoor to create database")
+	createDatabase := flag.Bool("createDatabase", true, "true if you need casdoor to create database")
```

Comment:
Why not add this to conf or env？

Reply:
@sagilio Why not automatically create the database if it doesn't exist, but configure it here? I think the auto-created behavior would be better.